prow/plugins/cla/cla.go

```diff
@@ -241,8 +240,8 @@ func handleComment(gc gitHubClient, log *logrus.Entry, e *github.GenericCommentE
 
 	for _, status := range combined.Statuses {
 
-		// Only consider "cla/linuxfoundation" status.
-		if status.Context == claContextName {
+		// Valid status values are cla/linuxfoundation and EasyCLA
+		if contains(cc.CLAContextNames, status.Context) {
```

Comment:
Can we rewrite this so that it iterates through statuses to determine CLA yes/no first, and then once outside the for loop, applies labels accordingly?  I'd like to avoid wasting GitHub tokens on label apply/remove calls while we decide which status wins if they start competing.

Reply:
Is `combined.Statuses` sorted at this point? or could we be walking these in random order?

Reply:
@spiffxp Can you please elaborate on competing statuses? Going by the existing code `if status.Context == claContextName`, I assumed the `for loop` should not continue if it is not one of `cla/linuxfoundation` or `EasyCLA`

Or are you saying this entire `for loop `should be rewritten such that applying of `ClaNo` or `ClaYes` should be outside the for loop?
Thanks.

Reply:
> Is combined.Statuses sorted at this point? or could we be walking these in random order?

@mrbobbytables Do you know about the sort order of `combined.Statuses`? Or rather, how can I check / verify the order?

Reply:
>Can you please elaborate on competing statuses?

I missed the `break` down below.  You're right, one of them will win.  But will it be the same one every time?  And how will I know which one that is?

Reply:
I removed the `break` statement so that all contexts will be processed.